person/src/Person.java

```diff
@@ -0,0 +1,86 @@
+
+
+/**
+ * Class Person
+ */
+public class Person {
```

Comment:
@tatevikb I will suggest to have "BasePerson" as it acts like base class for all people